docs/self-managed/platform-deployment/helm-kubernetes/platforms/redhat-openshift.md

```diff
@@ -258,3 +258,73 @@ Now, when installing the chart, you can do so by running the following:
 helm install <RELEASE_NAME> camunda/camunda-platform --skip-crds \
     -f values.yaml -f openshift.yaml --post-renderer ./patch.sh
 ```
+
+## Configuring Ingress using routes for Zeebe Gateway
+
+The Ingress on OpenShift works slightly different from the Kubernetes default. The mechanism is called [routes](https://docs.openshift.com/container-platform/4.10/networking/routes/route-configuration.html).
+
+To use these routes for the Zeebe Gateway, configure this through Ingress as well.
+
+### Alternatives
+
+An alternative is to install the Ingress Controller of choice and use this instead; for example, [NGINX](https://www.redhat.com/en/blog/using-nginx-ingress-controller-red-hat-openshift).
+
+### Prerequisite
+
+As the Zeebe Gateway uses `gRPC` (which relies on `HTTP/2`,) this [has to be enabled](https://docs.openshift.com/container-platform/4.10/networking/ingress-operator.html#nw-http2-haproxy_configuring-ingress).
+
+### Required steps
+
+1. Provide [TLS Secrets](https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets) for the Zeebe Gateway, the [Cert Manager](https://docs.openshift.com/container-platform/4.10/security/cert_manager_operator/index.html) might be helpful here:
+
+- One issued to the Zeebe Gateway Service Name. This must use the [pkcs8 syntax](https://www.openssl.org/docs/man3.1/man1/openssl-pkcs8.html) as netty only supports this, referenced as **Service Certificate Secret** or `<SERVICE_CERTIFICATE_SECRET_NAME>`.
```

Comment:
By "netty" do you mean Kubernetes here?

Reply:
No, netty is a server framework that is running inside zeebe under the hood and is also responsible serving cerificates.

At second glance, this is a very technical detail. What do you think about just leaving out this half sentence?

Reply:
Okay. If this detail isn't necessary, I would remove it. Otherwise, I would just briefly note what netty is here since this is its first reference in the doc.